dmoj/cptbox/tracer.py

```diff
@@ -223,6 +244,8 @@ def wait(self) -> int:
                 raise RuntimeError('failed to spawn child')
             elif self.returncode == PTBOX_SPAWN_FAIL_SETAFFINITY:
                 raise RuntimeError('failed to set child affinity')
+            elif self.returncode == PTBOX_SPAWN_FAIL_LANDLOCK:
+                raise RuntimeError('landlock configuration failed')
```

Comment:
```suggestion
                raise RuntimeError('Landlock configuration failed')
```

Nit, here and in other user-facing strings as well as comments. The kernel docs refer to it as "Landlock" (despite "seccomp" being lowercased).